server/controllers/googleApiController.js

```diff
@@ -1,27 +1,65 @@
 const { google } = require("googleapis");
+const moment = require("moment");
 
+// get google authentication
 const oAuth2Client = new google.auth.OAuth2(
-  "871373961261-rjej65g97dc3o6jiuflq6s2gp5v9ptut.apps.googleusercontent.com",
-  "XauLtqJv2LbkiRd7nTMhRnQq",
-  "http://localhost:3000"
+  process.env.client_id, process.env.client_secret, process.env.redirect_uris
 );
+// load auth for getting tokens from google authentication code
 const oauth2 = google.oauth2("v2");
+// load calendar for getting events from google calendar
+const calendar = google.calendar({version: "v3",auth: oAuth2Client});
 
 module.exports = {
   authenticateUser: function (req, res) {
     oAuth2Client
       .getToken(req.query.code)
       .then(({ tokens }) => {
-        oAuth2Client.setCredentials({ token: tokens });
-        return oauth2.userinfo.get({ access_token: tokens.access_token });
+        oAuth2Client.setCredentials(tokens);
+        return oauth2.userinfo.get({access_token: tokens.access_token});
       })
       .then((userInfo) => {
-        oAuth2Client.setCredentials({
-          token: oAuth2Client.credentials.token,
-          userInfo: userInfo.data,
-        });
-        res.status(200).json(oAuth2Client.credentials);
+        oAuth2Client.setCredentials({...oAuth2Client.credentials,...userInfo.data});
+        //TO DO: create user or update user with the user profile and new tokens
+        return res.status(200).json(oAuth2Client.credentials);
       })
       .catch((err) => res.status(422).json(err));
   },
+
+  getAvailability: function (req, res) {
+    console.log(oAuth2Client.credentials);
+    let date = `${req.query.year}-${req.query.month}-${req.query.date}`;
+    let meetingLength = req.params.meetingLength;
```

Comment:
small nit picks 

if `date` and `meetingLength` never change, turn these into `const`